tests/integration/serializers/active-model-serializer-test.js

```diff
@@ -29,7 +29,7 @@ module('Integration | Serializer | ActiveModelSerializer', {
       application: ActiveModelSerializer,
       wordSmith: ActiveModelSerializer.extend({
         attrs: ['id', 'name'],
-        relationships: ['blogPosts']
+        relationships: ['blog-posts']
```

Comment:
@lolmaus I meant all of these pieces - like I said, it seems weird writing camelCase in string, but they directly correspond to model properties (just like the `attrs` hash). So, if we can't think of a better API (and I don't think referencing camelCased properties with kebab-case is better), I'd like to keep these camelCased.